nixos/modules/services/web-servers/apache-httpd/default.nix

```diff
@@ -647,14 +647,15 @@ in
       wwwrun.gid = config.ids.gids.wwwrun;
     };
 
-    security.acme.certs = mapAttrs (name: hostOpts: {
-      user = cfg.user;
-      group = mkDefault cfg.group;
-      email = if hostOpts.adminAddr != null then hostOpts.adminAddr else cfg.adminAddr;
-      webroot = hostOpts.acmeRoot;
-      extraDomains = genAttrs hostOpts.serverAliases (alias: null);
-      postRun = "systemctl reload httpd.service";
-    }) (filterAttrs (name: hostOpts: hostOpts.enableACME) cfg.virtualHosts);
+    security.acme.certs = mapAttrs' (name: hostOpts: (
```

Comment:
Good catch. The previous code here assumed the attribute name was the host name which is not always the case! Can you file a separate PR for this change? I'd like to get this fix into `master` as soon as possible as we have feature freeze coming up soon.